mukurtu-gitpod-site.composer.json

```diff
@@ -151,14 +151,6 @@
                 "type:drupal-custom-theme"
             ]
         },
-        "patches": {
-            "drupal/message_ui": {
-                "Fix installation": "https://git.drupalcode.org/project/message_ui/-/merge_requests/2.diff"
-            },
-            "drupal/paragraphs": {
```

Comment:
Hi @nick-deer This looks like it removes the patch installation from gitpod. I think we probably need this in order for the patch to apply. I added the patch to the branch in an earlier commit 

https://github.com/MukurtuCMS/Mukurtu-CMS/blob/ba2a9a288f8b1ab0383e287049a10d86dcb062fb/patches/paragraphs-118.patch

Reply:
Oh my gosh you're right I'm sorry I missed that! Let me revert my changes on the template repo and I'll merge this PR and PR 604